lib/knex.js

```diff
@@ -54,6 +54,11 @@ function Knex(config) {
   if (config.userParams) {
     newKnex.userParams = config.userParams;
   }
+
+  if (config.defaultSchema) {
+    newKnex.withSchema(config.defaultSchema);
```

Comment:
As far as I can tell using `withSchema` on its own like this doesn't prepend the schema. At least not when printing out statements with debug on .